tests/testthat/test-screenFiles.R

```diff
@@ -17,7 +17,7 @@ context("Passes everything")
 test_that("PassEverything", {
   screeningOutput <- testOther("../../tests/shinytest/test-data/passes_everything.csv")
 
-  expect_equal(screeningOutput$results %>% filter(!(result %in% c("PASS", "PASS WITH NOTE"))) %>% nrow(), 0)
+  expect_equal(screeningOutput$results %>% filter(!(result %in% c("PASS", "PASS WITH NOTE", "IGNORE"))) %>% nrow(), 0)
```

Comment:
Ah, that's a hangover from when I was figuring out what was going on. i.e. there wasn't any LSIP data in the passes everything file, so it threw an ignore for that test. I've since added an LSIP level to the passes everything file and thought I'd removed that IGNORE in the filter.